website/docs/reference/change-requests.mdx

```diff
@@ -10,171 +10,104 @@ import VideoContent from '@site/src/components/VideoContent.jsx';
 
 :::note Availability
 
 **Plan**: [Enterprise](https://www.getunleash.io/pricing) | **Version**: `5.4+`
 
 :::
 
-Changes to project [segments](segments) (as opposed to global segments) also go through the change request process. This is to prevent a backdoor in the change request process.
-
-Since projects segments are not environment specific and change requests are always environment specific we allow to attach segment change to any environment with change requests enabled.
-When you make changes though the Change Request UI it will automatically select first environment with change requests enabled, giving priority to [production](environments#environment-types) environments.
-
-Changes to segments can be only circumvented by admin users through the API calls.
-
-## Change Request Preview Playground
-
-To verify that a change request is correct, you can preview the result of change request's application in a change request  [playground](playground). 
-
-![Change request preview](/img/change-request-preview.png)
-
-From the change request overview page, go to the corresponding [playground](playground) and evaluate all your flags in the project and environment that your change request applies too.
-
-![Change request playground evaluation](/img/change-request-playground-evaluation.png)
+Changes to project [segments](/reference#segments), unlike global segments, also go through the change request process.
 
-[Unleash context](playground#the-unleash-context) can be adjusted in the same way as in a regular playground, but the project and environment cannot be changed as they are derived from the change request itself.
-Once the evaluation results confirm the changes in your change request are correct, go back to the change request overview and proceed with the approval or rejection.
+Because project segments are not environment-specific while change requests are, Unleash allows you to attach segment changes to any environment where change requests are enabled. By default, Unleash selects the first available environment with change requests enabled, prioritizing [production](/reference/environments#environment-types) environments.
 
+Only Admin users can bypass the change request process for project segments through API calls.
 
-Change request preview simulates the application of changes in a non-persistent transaction. 
-You still need to apply the changes to persist them for the SDKs to see the changes.
-A change request can only be previewed when the change request is In Review, Approved, or Scheduled. 
-It cannot be previewed when the change request is in Draft, Applied, Cancelled, or Rejected status.
+## Change request preview
 
+To verify a change request, you can preview the changes in [Playground](/reference/playground) by clicking **Preview changes**. You can adjust [Unleash context](/reference/playground#the-unleash-context), but the project and environment remain fixed as they are determined by the change request.
```

Comment:
Ooh, is it just "Playground" or is it "_the_ Playground"? I've always used the definite article, I think, but maybe we don't want that🤔

Reply:
I think either works, but when we capitalize a feature/product name, to me it feels more natural to treat it as a proper noun.